.travis.yml

```diff
@@ -26,6 +26,8 @@ env:
   - TESTS="1" ODOO_REPO="OCA/OCB"
   - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="website_signup_legal_page_required"
   - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="website_signup_legal_page_required"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_seo_redirection"
```

Comment:
I understand your concerns, but I prefer to let it commented. People will know how to act in case of error.